features/manage-position/Withdraw.tsx

```diff
@@ -245,13 +243,13 @@ const Deposit = () => {
           </Typography>
           <ul style={{ fontSize: 16 }}>
             <li>
-              <strong>"Fast" withdrawal: </strong>Instantly withdraw collateral
-              until your positions collateralization ratio is equal to the
-              global collateralization ratio. For your position you can
-              instantly withdraw {fastWithdrawableCollateral} {collSymbol}.
+              <strong>"Fast" withdraw: </strong>Instantly withdraw collateral
```

Comment:
Like I did in the `contract-state/GeneralInfo` feature, can you link to the docs on Fast vs Slow withdrawals? Be sure to add the docs links to `utils/getDocLinks.ts`.